conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -20554,3 +20554,15 @@ msgid ""
 "A new revision was created: Merge between Revision #%(r1)d and Revision "
 "#%(r2)d"
 msgstr ""
+
+#: lms/templates/widgets/cookie-consent.html
```

Comment:
@cahrens, ok, I will then do a ```git checkout upstream/master conf/locale/en/LC_MESSAGES/django.po``` ? Out of interest, what is the correct approach when adding localisation/RTL support. Do these strings (that appear in the cookie-consent banner) need to be added to transifex first? I'm really new to EdX/internationalisation and your guidance would be much appreciated.